packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -195,19 +202,41 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
     return removeHtmlPTags(previewString);
   };
 
+  // if chat changes, update the internal state to match
+  useEffect(() => {
+    setChatInternal(chat);
+  }, [chat]);
+
+  // enrich the chat if necessary
+  useEffect(() => {
+    if (chatInternal.id && (!chatInternal.chatType || !chatInternal.members)) {
+      const provider = Providers.globalProvider;
+      if (provider && provider.state === ProviderState.SignedIn) {
+        const graph = provider.graph.forComponent('ChatListItem');
+        const load = (id: string): Promise<Chat> => {
+          return loadChatWithPreview(graph, id);
+        };
+        load(chatInternal.id).then(
+          c => setChatInternal(c),
+          e => error(e)
+        );
+      }
+    }
+  }, [chatInternal]);
+
   return (
     <Button
       className={styles.chatListItem}
       onClick={() => {
-        onSelected(chat);
+        onSelected(chatInternal);
       }}
     >
       <div className={styles.profileImage}>{getDefaultProfileImage()}</div>
       <div className={styles.chatInfo}>
-        <h3 className={styles.chatTitle}>{inferTitle(chat)}</h3>
-        <p className={styles.chatMessage}>{enrichPreviewMessage(chat.lastMessagePreview)}</p>
+        <h3 className={styles.chatTitle}>{inferTitle(chatInternal)}</h3>
```

Comment:
After merging latest, when the Chat is change, it resets and that temporary reset state can be seen visually until the load is complete. Should we address that now? 
<img width="379" alt="image" src="https://github.com/plasne/microsoft-graph-toolkit/assets/10041279/31d2362b-f328-4099-bc9b-caa607585030">